### DISCUSSION TITLE
Audio disturbances unless mic off in ASIO4ALL

### DISCUSSION THREAD

**Discussion by rldekkers** [jamulussoftware/jamulus#1673] [Technical Support & Help]
**The problem**
Enabling the internal laptop mic or headphone mic in ASIO4ALL causes audio disturbances. The audio becomes a grainy noise that is barely recognizable, and the audio regularly becomes (nearly) silent for about a second. When both mics are disabled, the sound is good. When a mic is enabled in ASIO4ALL, but muted in Jamulus, audio disturbances still occur. 

**Screenshot**
The items next to the orange dots are the headphone mic and the internal mic (in that order).
<img src="https://user-images.githubusercontent.com/5004525/117765190-e2023500-b22d-11eb-8455-2c5247eacbef.png" alt="drawing" width="400"/>

**Details**
ASIO4ALL v2.15 Beta 1
Jamulus 3.7.0
Windows 10 28H2
Surface Pro 5
Realtek HD Audio (SST)

Any advice would be appreciated :)

**Comment by dcorson-ticino-com** [comment]
Hi Piefert,
My recommendation is, to not use ASIO4ALL at all.  Unfortunately that means buying an audio interface and mic, but I have never gotten ASIO4ALL to work.  Horrible sound when it did something and it crashed the computer often even when not using sound.  Simply uninstall ASIO4ALL and the crashes stopped.
Here a link to my recommendations in another case:  https://github.com/jamulussoftware/jamulus/discussions/1392#discussioncomment-566362

**Reply by rldekkers** [reply]
Thanks for your recommendation. I was hoping to get away without having to purchase anything, but perhaps I should see it as a worthwhile long-term investment.

**Comment by gilgongo** [comment]
You could try using Beta 2 version and see if that makes a difference? 

https://github.com/jamulussoftware/assets/tree/main/ASIO4ALL/v2.15

**Reply by rldekkers** [reply]
Thanks for your suggestion, I will try that.

**Comment by henkdegroot** [comment]
> When a mic is enabled in ASIO4ALL, but muted in Jamulus, audio disturbances still occur.

Please note that the "mute myself" is NOT disabling the mic, so it makes sence that the result is the same.
It might be worth to check in windows sound recording settings if there is an "input boost" option available and enabled or not.
